### PR TITLE
numa.numa_stress: Get tmpfs_write_speed of the host before test

### PR DESCRIPTION
**Pull request by huangyum**
Get the speed of "dd if=/dev/urandom of=tmpfs" on host, in order to get appropriate timeout.

ID: 1404125
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by huangyum** [comment]
@xutian @MiriamDeng , please help review this pr, the former PR #882 can not be reopened.

File: qemu/tests/numa_stress.py [hunk 1]
```diff
@@ -35,6 +35,25 @@ def max_mem_map_node(host_numa_node, qemu_pid):
     return (node_map_most, memory_sz_map_most)
 
 
+def get_tmpfs_write_speed():
+    """
+    Get the tmpfs write speed of the host
+    return: The write speed of tmpfs, the unit is kb/s.
+    """
+    utils.run("mount -t tmpfs none /tmp")
+    dd_cmd = "dd if=/dev/urandom of=/tmp/test_speed bs=1k count=1024"
```
Comment (xutian) [inline qemu/tests/numa_stress.py]:
```
output = utils.run("dd_cmd = "dd if=/dev/urandom of=/tmp/test_speed bs=1k count=1024")
try:
    speed = re.search("\s([\w\s\.]+)/s", output.stderr, re.I).group(1)
    return float(utils_misc.normalize_data_size(speed, 'K', 1024))
except Exception:
      return 3072
finally:
      os.remove("/tmp/test_speed")
```

File: qemu/tests/numa_stress.py [hunk 2]
```diff
@@ -35,6 +36,23 @@ def max_mem_map_node(host_numa_node, qemu_pid):
     return (node_map_most, memory_sz_map_most)
 
 
+def get_tmpfs_write_speed():
+    """
+    Get the tmpfs write speed of the host
+    return: The write speed of tmpfs, the unit is kb/s.
+    """
+    utils.run("mount -t tmpfs none /tmp")
```
Comment (xutian) [inline qemu/tests/numa_stress.py]: don't use /tmp driectly, some process write temp file in /tmp when avocado running,  remount maybe cause random test failed.
suggest to make a new dir in /tmp folder, then mount tmpfs to it.

**Comment by huangyum** [comment]
@xutian Thanks, updated!

**Comment by xutian** [comment]
looks OK to me.

**Comment by xutian** [comment]
Merged, thanks @huangyum

**Pull request merged**